cli/cmd/completion.go

```diff
@@ -53,6 +37,60 @@ else
     complete -o default -o nospace -F __start_cortex cx
 fi
 `
-		fmt.Print(aliasText)
+
+var _completionCmd = &cobra.Command{
+	Use:   "completion SHELL",
+	Short: "generate shell completion scripts",
+	Long: `generate shell completion scripts
+
+to enable cortex shell completion:
+    bash:
+        add this to ~/.bash_profile (mac) or ~/.bashrc (linux):
+            source <(cortex completion bash)
+
+        note: bash-completion must be installed on your system; example installation instructions:
+            mac:
+                1) install bash completion:
+                   brew install bash-completion
+                2) add this to your ~/.bash_profile:
+                   source $(brew --prefix)/etc/bash_completion
+                3) log out and back in, or close your terminal window and reopen it
+            ubuntu:
+                1) install bash completion:
+                   apt update && apt install -y bash-completion  # you may need sudo
+                2) open ~/.bashrc and uncomment the bash completion section, or add this:
+                   if [ -f /etc/bash_completion ] && ! shopt -oq posix; then . /etc/bash_completion; fi
+                3) log out and back in, or close your terminal window and reopen it
+
+    zsh:
+        option 1:
+            add this to ~/.zshrc:
+                source <(cortex completion zsh)
+            if that failed, you can try adding this to ~/.zshrc above the source command:
```

Comment:
if that failed, you can try adding this line above the source command in ~/.zshrc: